client/App/App.js

```diff
@@ -22,6 +25,11 @@ export default class App extends Component {
          */}
         <PersistGate loading={null} persistor={persistor}>
           <RootScreen />
+          <Button title="Stop working" onPress={() => {
```

Comment:
can be one liner
`onPress={() => stopForegroundService()}`
or even
`onPress={stopForegroundService}`